hypothesis-python/tests/cover/test_stateful.py

```diff
@@ -1248,6 +1248,45 @@ def fail_fast(self, param):
     )
 
 
+@pytest.mark.parametrize(
+    "returned,repr_",
+    [
+        ("ret1", "a_0 = b_0 = a_1 = "),
+        (multiple(), ""),
+        (multiple("ret1"), "(a_0,) = (b_0,) = (a_1,) = "),
+        (multiple("ret1", "ret2"), "{a_0, b_0, a_1; a_2, b_1, a_3} = "),
```

Comment:
The present behaviour is that the multiple results are (slightly) randomly distributed among the multiple targets. I'm assuming this is the intended behaviour, so I used curly brackets for this case - which IMO draws attention to the unordered nature by looking set-like.

I've also assumed (without digging) that this only happens in the multi-multi case. Feel free to chip in, and also we might want to add something to the `multiple()` docstring.